<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorGlissando</class>
 <widget class="QWidget" name="InspectorGlissando">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="accessibleName">
   <string>Glissando Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="elementName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="text">
      <string>Glissando</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="resetText">
       <property name="accessibleName">
        <string>Reset Text value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="resetType">
       <property name="accessibleName">
        <string>Reset Type value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="type">
       <property name="accessibleName">
        <string>Type</string>
       </property>
       <item>
        <property name="text">
         <string>straight</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>wavy</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="text">
       <property name="accessibleName">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="resetShowText">
       <property name="accessibleName">
        <string>Reset Show Text value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="showText">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="accessibleName">
        <string>Show Text</string>
       </property>
       <property name="text">
        <string>Show Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>type</tabstop>
  <tabstop>resetType</tabstop>
  <tabstop>text</tabstop>
  <tabstop>resetText</tabstop>
  <tabstop>showText</tabstop>
  <tabstop>resetShowText</tabstop>
 </tabstops>
 <resources>
  <include location="../musescore.qrc"/>
 </resources>
 <connections/>
</ui>
